lib/shared/runtime-info.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @fileoverview Utility to get information about the execution environment.
+ * @author Kai Cataldo
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const path = require("path");
+const spawn = require("cross-spawn");
+const { isEmpty } = require("lodash");
+const log = require("../shared/logging");
+const packageJson = require("../../package.json");
```

Comment:
```suggestion
const { version } = require("../../package.json");
```

Reply:
We already have a function in the module scope called `version`, so I'm afraid this won't work. I also personally think having `packageJson.version` makes it immediately clear exactly what we're checking the version of.